apps/webapp/web/graphql/resolver.ex

```diff
@@ -30,9 +30,9 @@ defmodule Webapp.GraphQL.Resolver do
   end
 
   # TODO: Don't forget to remove this StarWars stuff after extracting for pwr2-docker
-  alias Webapp.GraphQL.StarWarsDB
+  alias StarWars.GraphQL.DB
```

Comment:
Please use `alias StarWars.GraphQL.DB, as: StarWarsDB`. Just `DB` might lead to further confusion.